File: app/src/main/java/com/example/app/data/api/APIService.java
package com.example.app.data.api;

import retrofit2.Call;
import retrofit2.http.Field;
import retrofit2.http.POST;

public class APIService {

    @POST("/public/api/login.php")
    @
    Call<Response> login(@Field("username") String username, @Field("password") String password) {
        return null;
    }
}

File: app/src/main/java/com/example/app/ApiService.kt
package com.example.app

import okhttp3.RequestBody
import retrofit2.Call
import retrofit2.http.Multipart
import retrofit2.http.POST
import retrofit2.http.Part


interface ApiService {
    @Multipart
    @POST("uploadAvatar.php")
    fun uploadFile(@Part file: Part?, @Part("file") name: RequestBody?): Call<*>?


}